clang/test/Driver/fat-lto-objects.c

```diff
@@ -41,6 +41,9 @@
 // CHECK-CC-NOLTO-SAME: -emit-obj
 // CHECK-CC-NOLTO-NOT: -ffat-lto-objects
 
+/// Disable fat LTO when it is overridden by -fno-fat-lto-objects
```

Comment:
You can delete this comment. This is an obvious case and we don't add similar comments elsewhere.